src/main/java/com/google/netpcapanalysis/servlets/TableServlet.java

```diff
@@ -10,40 +10,47 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
 
 @WebServlet("/data")
 public class TableServlet extends HttpServlet {
-
-  private static final String FILE_NAME = "file_1";
+  UserService userService = UserServiceFactory.getUserService();
   private PCAPDao data = new PCAPDaoImpl();
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-
-    String json = convertToJsonUsingGson(data.getPCAPObjects(FILE_NAME));
+    if (userService.isUserLoggedIn()) {
+    String json = convertToJsonUsingGson(data.getPCAPObjects(userService.getCurrentUser().getEmail()));
     response.setContentType("application/json;");
     response.getWriter().println(json);
+
+    } else {
+      response.sendRedirect("/login");
```

Comment:
is it mandatory to Login to View the site? If its not too hard  having a public option would be nice, so without logging in they can still view some PCAP files>